### PR TITLE
Feature/deployment to heroku ect

### LINKED ISSUE #6: Deploy on Heroku

### PR DESCRIPTION
**Pull request by ectweitmann**
## What Features are Being Added?
- Created `.env` 
- Set up production environment in `knexfile.js`
- Installed dotenv npm package
- Added script to run server in package.json

## What is Being Refactored?
- Refactored repo structure. 
  - Removed db repo to solve bug with heroku being unable to find migrations and seeds directories
- Changed environment connection from development to production in `queries.js`

## Where should the reviewer start?
- Start in knexfile.js to see the new production environment. 

## How should this be manually tested?
- Using our deployed link: `https://book-club-api-2110.herokuapp.com/api/v1/` try out the different end points on postman to see whether we are getting back the proper responses/error handling messages.

## What are the relevant tickets?
- Deploy to heroku!
## Any Other Notes/Questions/Problems?
- Way to go everyone, that was not easy and I'm proud of the way we all stuck through it and figured this out!

### CODE REVIEW

File: knexfile.js [hunk 1]
```diff
@@ -3,6 +3,7 @@
 /**
  * @type { Object.<string, import("knex").Knex.Config> }
  */
+require('dotenv').config();
```
Comment (cagallo) [inline knexfile.js:6]: Needed this!

**Review by cagallo — APPROVED** [review]
Great job Ethan, deployment was tough. So happy we got it. Thanks for all your hard work!

File: knexfile.js (hunk 1 above)
Comment (krdurrett) [inline knexfile.js:6]: good to know we will need to install dotenv and require it in the future!

File: package.json [hunk 2]
```diff
@@ -18,6 +19,7 @@
   },
   "homepage": "https://github.com/cagallo/book-club-api#readme",
   "dependencies": {
+    "dotenv": "^16.0.0",
```
Comment (krdurrett) [inline package.json:22]: thanks for installing this!

**Review by krdurrett — APPROVED** [review]
Nice work addind this dotenv.

**Pull request merged**